frontend/src/metabase/components/ExplicitSize/ExplicitSize.tsx

```diff
@@ -72,6 +75,12 @@ function ExplicitSize<T extends BaseInnerProps>({
         };
 
         this._printMediaQuery = window.matchMedia && window.matchMedia("print");
+        if (this.context) {
+          this._refreshMode = this.context as RefreshMode;
```

Comment:
This is a way to override `_refreshMode` when testing with Loki. This will help decrease the likelihood of flaky tests for components relying on `ExplicitSize` since this ensures we don't wait before calling `_updateSize`.

There were runs that flake without overriding `_refreshMode`, so I think this is necessary. Just to be clear, with or without this change, snapshot tests in this PR always pass locally.

Reply:
On second thought, I'm not sure if using `isLokiRunning` here would be more direct. Please let me know.